script/build.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+##
+# Run the application and content builds.
+###
+
+assetSource="local"
+buildtype="localhost"
+
+# Save the arguments to this script for later; they get drained in the following for loop
+args="$*"
+
+# Prepend env. to all args so they get passed to the Webpack config
+webpackArgs=$(echo -n "${args}" | sed -E 's/--([^ =]+)/--env.\1/g')
```

Comment:
Because nothing can be easy, right? In order for CLI args to get passed to `webpack`, they need to be `--env.whatever`. Documentation can be found [here](https://webpack.js.org/guides/environment-variables/).

Reply:
What's the reason for putting some logic in bash and some logic in node? Node has access to the command line arguments. You could use something like [`command-line-args`](https://github.com/75lb/command-line-args#readme) to define accepted arguments.

Reply:
We potentially could make this a `.js` script, but since it's in charge of orchestrating two separate yarn scripts, I decided to keep it in the shell. For what it's worth, this is only a temporary thing until we can remove the content build from `vets-website` entirely.